Pidgin/Parser.NoneOf.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+using Pidgin.ParseStates;
+
+namespace Pidgin
+{
+    public static partial class Parser
+    {
+        /// <summary>
+        /// Creates a parser which parses and returns a character as long as it does not match one of the specified characters.
+        /// </summary>
+        /// <param name="chars">A sequence of characters that should not be matched</param>
+        /// <returns>A parser which parses and returns a character that does not match one of the specified characters</returns>
+        public static Parser<char, char> NoneOf(params char[] chars)
```

Comment:
I think you are right that this is really unrelated to the Not changes and so I changed it up to not use `Not` and put the methods inside of the Parser.Char.cs file. I think AnyCharExcept seems to be the most descriptive.